breeze/breeze.py

```diff
@@ -553,3 +554,70 @@ def list_pledges(self, campaign_id):
         return self._request('%s/list_pledges?campaign_id=%s' % (
             ENDPOINTS.PLEDGES, campaign_id
         ))
+
+    def get_tags(self, folder=None):
+        """List of tags
+
+        Args:
+          folder: If set, only return tags in this folder id
+
+        Returns:
+          JSON response. For example:
+            [
+            {
+                "id":"523928",
+                "name":"4th & 5th",
+                "created_on":"2018-09-10 09:19:40",
+                "folder_id":"1539"
+            },
+            {
+                "id":"51994",
+                "name":"6th Grade",
+                "created_on":"2018-02-06 06:40:40",
+                "folder_id":"1539"
+            },
+            { ... }
+            ]"""
+
+
+        params = []
+        if folder:
+            params.append('folder_id=%s' % folder)
+        return self._request('%s/%s/?%s' % (ENDPOINTS.TAGS, 'list_tags', '&'.join(params)))
```

Comment:
I question whether the '/' before '?' is needed here. Seems superfluous. But this matches existing usage.